api_tests/dry/sanity.ts

```diff
@@ -63,7 +63,7 @@ setTimeout(async function() {
                 "content-type",
                 "application/problem+json"
             );
-            expect(res.body.title).to.equal("Swap not supported.");
+            expect(res.body.title).to.equal("Invalid body.");
```

Comment:
:upside_down_face: Test names are slightly misaligned with what the tests do now.